cadquery/occ_impl/shapes.py

```diff
@@ -1072,14 +1101,51 @@ def tessellate(
 
         return vertices, triangles
 
-    def _repr_html_(self):
+    def toVtkPolyData(
+        self, tolerance: float, angularTolerance: float = 0.1, normals: bool = True
```

Comment:
No default for the tolerance here?

Reply:
Good point - I'll add 1e-3

Reply:
Actually I just followed the conventions of mesh and tessellate. I think the rational is - this value is use case dependent and it is a low level API assuming that a potential user will be able to judge himself which tol is needed.